lib/run_loop/plist_buddy.rb

```diff
@@ -233,7 +233,9 @@ def build_plist_cmd(type, args_hash, file)
             raise(ArgumentError, "expected '#{value_type}' to be one of '#{allowed_value_types}'")
           end
           value = args_hash[:value]
-          unless value
+          if value_type == 'bool'
```

Comment:
I'm not sure about this piece of code. Why the check for if value_type is bool? The double negation ensures that value will always be a bool value, even if it was initially nil. If value can be any value, for example: Bool or String, then I would argue this is overuse of the dynamic typing and super confusing.

So my suggestion would be to rewrite to:
```
value = !!value
unless value
  raise ....
end
```
Of course, unless could also be written as  `If !value`. I just don't like the `!` but well 🤷‍♂️

Reply:
actually could even be 
```
value = !!value
raise .... unless value
```

Reply:
Ah, take it back, I see that this is a plist value. Should read more context first I guess.

Reply:
the problem that I was trying to fix is exactly `unless value`, because this will raise in value is false or nil, which is a valid option if you pass a boolean

Reply:
I also understand that making this a bit confusing code even more confusing, is not a best option. But I will think about refactoring it in the upcoming PRs

Reply:
> because this will raise if value is false or nil, which is a valid option if you pass a boolean

An obvious bug.  :)